changelog/5414.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+:meth:`sunpy.map.GenericMap.draw_limb` no longer plots or returns the hidden
+part of the limb if it does not have any points.
```

Comment:
```suggestion
When the limb is entirely visible, :meth:`sunpy.map.GenericMap.draw_limb` no longer plots an invisible patch for the hidden part of the limb and now returns `None` instead of the invisible patch.
```